tests/helpers.py

```diff
@@ -854,6 +858,99 @@ def create_pods(pvc_objs, pod_factory, interface, pods_for_rwx=1, status=""):
     return pod_objs
 
 
+def create_build_from_docker_image(
+        image_name,
+        install_package,
+        namespace,
+        source_image='centos',
+        source_image_label='latest'
+):
+    """
+    Allows to create a build config using a Dockerfile specified as an argument
+    For eg., oc new-build -D $'FROM centos:7\nRUN yum install -y httpd',
+    creates a build with 'httpd' installed
+
+    Args:
+        image_name (str): Name of the image to be created
+        source_image (str): Source image to build docker image from,
+        Defaults to Centos as base image
+        namespace (str): project where build config should be created
+        source_image_label (str): Tag to use along with the image name,
+        Defaults to 'latest'
+        install_package (str): package to install over the base image
+
+    Returns:
+        (obj): Returns the image object
```

Comment:
Please add the exceptions, also will be good if you can add more comments in the code.